src/displayapp/LittleVgl.cpp

```diff
@@ -255,6 +255,11 @@ void LittleVgl::CancelTap() {
   }
 }
 
+void LittleVgl::ClearTap() {
+  touchPoint = {-1, -1};
```

Comment:
We could though I aimed to avoid refactoring the entire LittleVgl class
LVGL needs to read ints and the conversion could be done in `GetTouchPadInfo`